src/python/grpcio/commands.py

```diff
@@ -58,6 +63,117 @@ class CommandError(Exception):
   """Simple exception class for GRPC custom commands."""
 
 
+# TODO(atash): Remove this once PyPI has better Linux bdist support
+def _linux_target_egg_name(grpc_version):
+  """Get gRPC's custom made-up target names for Linux."""
+  assert sys.platform.startswith('linux')
+  version = '.'.join(map(str, sys.version_info[0:2]))
```

Comment:
How about naming this field `python_version`, just to be extra clear?

Also would its value be more accessibly expressed as `'{}.{}'.format(sys.version_info[0], sys.version_info[1])`? I'm not actually sure; it's up to you.